source/scss/_pf-type.scss

```diff
@@ -76,19 +72,19 @@ h6{
 }
 
 .pf-vertical-rhythm h1 {
-  @include pf-heading-styles($weight: $pf-font-weight-light, $size: $font-size-h1, $margin-top: $pf-headings-margin-top, $margin-bottom: $pf-typography-margin-bottom);
+  @include pf-heading-styles($weight: $pf-font-weight-light, $size: $font-size-h1, $margin-top: $pf-headings-margin-top, $margin-bottom: $headings-margin-bottom);
```

Comment:
should we ditch "pf" for the pf-headings-margin-top, or do we keep that because it's not overriding a BS variable?